components/productCard.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import styles from "../styles/productCard.module.css";
+
+interface Product {
+  id: number;
+  category: string;
+  name: string;
+  price: number;
+}
+
+interface ProductCardProps {
+  product: Product;
+}
+
+const ProductCard: React.FC<ProductCardProps> = ({ product }) => {
+  const productImages = [
```

Comment:
[Not a blocker] As this array serves as a constant data piece, we could perhaps name it as `PRODUCT_IMAGES` following the capital letter snake case and place it outside the component function to clearly show that is does not relate to the component code logic?